app/controllers/proficient_projects_controller.rb

```diff
@@ -49,7 +49,8 @@ def index
   end
 
   def requests
-    @order_item_waiting_for_approval = OrderItem.all.waiting_for_approval
+    @order_item_waiting_for_approval =
+      OrderItem.all.waiting_for_approval.order(updated_at: :desc).reverse_order
```

Comment:
Why reverse order on an order ??

Reply:
to order the items from oldest to newest (top to bottom), that way admins can find the project that's been waiting for review for the longest at the top of the page

Reply:
can't you do updated_at: :asc then?